modules/config/variables.tf

```diff
@@ -0,0 +1,26 @@
+# REQUIRED parameters
+
+variable "resource_name_prefix" {
+  description = "Resource name prefix used for tagging and naming AWS resources."
+  type        = string
+}
+
+# OPTIONAL parameters
+
+variable "graphdb_admin_password" {
+  description = "Password for the 'admin' user in GraphDB."
+  type        = string
+  default     = "s3cret"
```

Comment:
I think you mentioned something about generating these. Anyway, I think it might be best if we don't put defaults in the submodules - wdyt?

Reply:
TBH, I cannot decide which is best, if we should assing defaults for each module or not.